lib/autoload/send_file.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 module SendFile
+  MULTIPART_CHUNK_SIZE = 10.megabytes
```

Comment:
> Part size	5 MiB to 5 GiB. There is no minimum size limit on the last part of your multipart upload.  
> Maximum number of parts returned for a list parts request	1000  

https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html

Any reason why 10 MB was chosen? Or just picked a value?  
This also means that we can only upload 1000 * 10 MB which is 10 GB file size max.